client/client_pool/include/client/client_pool/concord_client_pool.hpp

```diff
@@ -45,6 +45,9 @@ enum SubmitResult {
   Acknowledged,  // The request has been queued for submission
   Overloaded,    // There is no available client to the moment to process the
   // request
+  Invalid_Argument,
+  Timed_Out,
+  Client_Unavailable,
```

Comment:
`InvalidArgument` without the underscore - same for the others.
Also, can you please add a comment to `ClientUnavailable` and provide more details.

Reply:
Done